### PR TITLE
feat: audit-secureblue: add suggestions for new perm checks

### PR DESCRIPTION
**Pull request by spaceoden**
the newly added host-os:ro and device=all checks should offer suggestions as well

### CODE REVIEW

**Comment by TheHooly** [comment]
I'd highly recommend adding device=dri when device=all is revoked. Otherwise many apps will use software rendering or stop working entirely.

**Comment by spaceoden** [comment]
@TheHooly I added a recommendation to enable device=dri

File: files/justfiles/secureblue.just [hunk 1]
```diff
@@ -700,6 +702,11 @@ audit-secureblue:
                 warnings+=("> $f has device=all permission,")
                 warnings+=("> granting access to GPU, input devices, raw USB, and virtualization,")
                 warnings+=("> and introducing a vector for sandbox escapes!")
+                warnings+=("> To remove it use Flatseal or run:")
+                warnings+=("> 'flatpak override -u --nodevice=all $f'")
+                warnings+=("> We also recommend you enable device=dri to grant GPU access")
```
Comment (RoyalOughtness) [inline files/justfiles/secureblue.just]: (nit) "to grant GPU access, if needed by the application"

Comment (spaceoden) [reply files/justfiles/secureblue.just]: added, but said "app" instead of "application" to keep line width <80

**Review by RoyalOughtness — APPROVED** [review]

**Review by RKNF404 — APPROVED** [review]

**Review by TheHooly — APPROVED** [review]

**Pull request merged**